aggregations-and-analytics/12_lag/12_lag.md

```diff
@@ -0,0 +1,43 @@
+# 12 Lag
```

Comment:
What do you think we can use as an attractive title for the recipe? I like how Microsoft uses 'Accesses data from a previous row in the same result set without the use of a self-join' but that's a bit too long 😅

Reply:
Yeah, Lag is a bit "dry", what about one between the following

* Retrieve item's previous value without self-join
* Retrieve history with Lag
* Go back in item's history with Lag
* Compare item's history with Lag

Reply:
I like `Retrieve previous row value without self-join`. WDYT?

Reply:
Perfect!